### DISCUSSION TITLE
mtDNA genotyping

### DISCUSSION THREAD

**Discussion by AnjaliC4** [stuart-lab/signac#805] [Q&A] [answered]
Hi, I have a basic question. Do you think it is possible to use mitochondrial genotyping for single-nucleus ATAC-seq? I guess under the assumption that most mtDNA detected in a nucleus is from the same cell?
Thanks

**Comment by caleblareau** [comment, marked as answer]
@AnjaliC4 mitochondrial DNA is strictly located in the cytoplasm, meaning that single- nucleus ATAC-seq data is NOT appropriate for mtDNA genotyping analysis. 

You may get reads that associate with mitochondrial DNA after read alignment, but in our experience, these are not suitable or useful for any downstream inferences as they likely originated from either a) NUMTs or b) non-specific mitochondria that "stick" to nuclei (but not necessarily the one that is captured in the droplet). 

Unless you are proceeding with a protocol that specifically retains the cytoplasm (e.g. mtscATAC-seq), this is not recommended.
